lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -433,6 +433,48 @@ func (c *Client) start() {
 						return
 					}
 				}
+			case tdp.SharedDirectoryListResponse:
+				if c.cfg.AllowDirectorySharing {
+					// fsoList is memory handled by Go, and will be freed
+					// by the garbage collector automatically sometime after
+					// this code block ends.
```

Comment:
I'm curious the reasoning behind this comment. Any time you `make` a slice in Go the memory is handled by Go. Was there some reason we felt the need to clarify that here?

Reply:
This was the first time I'd needed to pass something non-trivial across the FFI; this is basically me baby-talking myself through the memory management logic. It can be removed from the production code.